executor/runtime/docker/docker_ssh.go

```diff
@@ -76,7 +76,7 @@ PasswordAuthentication no
 #GSSAPIAuthentication no
 #GSSAPICleanupCredentials yes
 
-X11Forwarding no
+X11Forwarding yes
```

Comment:
Hmm, I think this config was copy/pasted to be consistent with the [EC2 base OS config](https://stash.corp.netflix.com/projects/BASEAMI/repos/nflx-sshd-config/browse/root/etc/ssh/sshd_config#73).

Should we synchronize this change with base OS?

Reply:
Yea sure, I can do that.
As long as security signs off on it :)

Reply:
cc @russell-lewis in case there's any context about why we've had this off historically?

Reply:
We weren't running X11 anywhere at the time.